protected/humhub/modules/search/jobs/RebuildIndex.php

```diff
@@ -48,7 +41,7 @@ public function run()
      */
     public function getTtr()
```

Comment:
We can also remove this method. Since all LongRunningJobs should have a large Ttr.